.gitignore

```diff
@@ -1,17 +1,20 @@
 node_modules
 *.log
 .idea
+
+package-lock.json
```

Comment:
I guess that makes sense but it certainly feels weird to be ignoring a file that npm tells you to check in. We should open an issue about this I guess